pkg/tasks/apt.go

```diff
@@ -0,0 +1,72 @@
+package tasks
+
+import (
+	"fmt"
+	"strings"
+)
+
+func init() {
+	t := registerTaskDefinition("apt")
+	t.name = "Apt"
+	t.parser = parserApt
+}
+
+func parserApt(config *taskConfig, task *Task) error {
+	packages, err := config.getListOfStrings()
+	if err != nil {
+		return err
+	}
+
+	if len(packages) == 0 {
+		return fmt.Errorf("no Apt packages specified")
+	}
+
+	task.header = strings.Join(packages, ", ")
+
+	task.addAction(&aptInstall{packageNames: packages})
+
+	return nil
+}
+
+type aptInstall struct {
+	packageNames        []string
+	missingPackageNames []string
+}
+
+func (a *aptInstall) description() string {
+	return ""
```

Comment:
Maybe `Install selected packages using apt` ?

Reply:
Returning en empty string is a lousy way to say : "don't display a task_action header".
When there is only one action, it is obvious since it's the same as the **task**